.github/workflows/linux.yml

```diff
@@ -0,0 +1,72 @@
+# Please keep all OS workflows identical except the name and the matrix config
```

Comment:
The Linux build finishes after ~10 minutes whereas Windows and macOS take 25+ minutes. If they share a workflow then you can't download artifacts for any of them until they **all** complete, so 25 minutes for Linux instead of just 10 minutes.

Reply:
I don't really think that this is a problem except for the case when testing the AppImage process.

Reply:
It's most likely that linux build faster because currently it's build in debug mode (because of error in cmake configuration script)